src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationContainer/ManageReputationContainer.tsx

```diff
@@ -163,6 +169,9 @@ const ManageReputationContainer = ({
               isVotingExtensionEnabled={isVotingExtensionEnabled}
               back={() => callStep(prevStep)}
               ethDomainId={ethDomainId}
+              verifiedUsers={
+                (isWhitelistActivated && verifiedUsers) || subscribedUsers
+              }
```

Comment:
@danbr Hmm, interesting. I am unable to reproduce. With a contributor and a watcher, I add the watcher to the address book, then deactivate the address book via the toggle switch, and after a browser refresh on the ColonyHome page I can see both users in the Award/Smite and CreatePayment dialogs. 

This is where I implement the check for whether the whitelist is activated. It should pass the form (and therefore the Single User Picker) all users (i.e `subscribedUsers`) if `isWhitelistActivated` is false (even if there is a member added to the address book).  Maybe you can spot something I'm overlooking?

Reply:
@willm30 Finally, I deleted my local branch and now its behaving as expected, thankfully!! I'll review the code next....